recipe/meta.yaml

```diff
@@ -13,16 +13,16 @@ source:
   '{{ hash_type }}': '{{ hash_value }}'
 
 build:
-  number: 1
+  number: 2
```

Comment:
I don't think you need to rebuild this to do the swicth, there should be no difference from the user pov

Reply:
Thanx, but I'll leave that as "better safe than sorry".